app/views/annotation_categories/_annotation_category_edit.html.erb

```diff
@@ -19,6 +20,16 @@
         <%= f.label :annotation_category_name,
                     AnnotationCategory.human_attribute_name(:annotation_category_name) %>
         <%= f.text_field :annotation_category_name, rows: '3' %>
+        <%= f.label :flexible_criterion_id,
+                    AnnotationCategory.human_attribute_name(:flexible_criterion) %>
+        <%= f.collection_select :flexible_criterion_id,
```

Comment:
Why ta_visible? Also, possible repetition of assignment specification for the criterion.

Reply:
We can be less restrictive, remove this specificity.